rust/data/presets.json

```diff
@@ -106,24 +108,22 @@
     "tech": [
       "canTrickyJump",
       "canDelayedWalljump",
-      "canLateralMidAirMorph",
       "canSpringBallBounce",
       "canSuitlessLavaDive",
       "canHitbox",
       "can3HighMidAirMorph",
       "canTurnaroundAimCancel",
       "canTrickyUseFrozenEnemies",
       "canMetroidAvoid",
-      "canMaridiaTubeClip",
       "canGrappleJump",
+      "canSnailClimb",
```

Comment:
On the fence about this one. It seems easier than most of the tech in `Hard` to me, but people struggle with it.

Reply:
Maybe a better way is to make versions of snailclimb strats with HiJump or springball.  And that would move itemless snail climb to notable.  Either way, it's not fun to see aqueduct early in maprando seeds with no items.  
Reaching top left door is the standout problem.  But I could see right door being changed too.